src/Vite.AspNetCore/TagHelpers/ViteTagHelper.cs

```diff
@@ -112,10 +112,25 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
 			// If the Vite script was not inserted, it will be prepended to the current element tag.
 			if (ViteStatusService.IsMiddlewareRegistered && !this._status.IsDevScriptInserted)
 			{
+				var baseUrl = CombineUri(this._viteOptions.GetViteDevServerUrl(), this._viteOptions.Base ?? string.Empty);
```

Comment:
If a base path is specified (example: `/dist/`) then the `viteClientUrl` will be something like `http://localhost:5173/dist/@vite/client` as i can see in the next line.

Reply:
If you set a basepath in the vite.config.ts the basepath of the modules are located there as well. This is why this change is needed. We were running vite along a different basepath.

Reply:
I see. It's okay then 👌